openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java

```diff
@@ -3558,10 +3558,16 @@ public EntityUpdater(
       this.original = original;
       this.updated = updated;
       this.operation = operation;
-      this.updatingUser =
-          updated.getUpdatedBy().equalsIgnoreCase(ADMIN_USER_NAME)
+      User originalUser =
```

Comment:
for every call are we going ot make a user_entity call? 
that will affect the performance severely